### ISSUE TITLE
Feature Request: use ERB instead of HAML

### ISSUE THREAD

**Issue by olleolleolle** [railsadminteam/rails_admin#3173]
TL;DR: This Issue is about using the preinstalled ERB instead of HAML for templating.

@mroderick and I are Rails Admin users. We'd volunteer to try to make this happen, provided this Issue is accepted.

### The landscape

- Rails Admin was created in a cultural moment when HAML was everywhere
- Time passed, HAML lost its dominance
- People have asked "Can we lose HAML?" before, and then was not the time
- Rails Admin 2.0 could have fewer opinions. Perhaps now is the time.

### Reasons to allow people not to use HAML

- Users and maintainers get to have **fewer gem dependencies**
- New users won't have to learn HAML, everyone has to learn ERB

## Solution proposal

- before 2.0, change all templates that come with Rails Admin to ERB
- have a tool-assisted workflow, to make this feasible
    - https://github.com/danchoi/herbalizer
- try to keep people's apps working, with their existing template overrides

## Elsewhere

* https://thoughtbot.com/blog/fashion-runway-erb-vs-haml
* https://opensoul.org/2011/11/30/haml-the-unforgivable-sin/
* https://speakerdeck.com/klaustopher/erb

**Comment by Andrew-Max** [comment]
I'm actually dealing with this right now. One issue though is that herbalizer cannot fully convert this haml. 

Sometimes it data attributes are passed to elements in a block and herbalizer cannot parse this.

**Comment by jasnow** [comment]
Does this help? https://haml2erb.org/

**Comment by mroderick** [comment]
> Does this help? https://haml2erb.org/

`herbalizer` is the tool that powers that website

**Comment by RocKhalil** [comment]
I've opened this PR for this purpose: https://github.com/sferik/rails_admin/pull/3191

If you can help with the tests, it would be perfect 😃

**Comment by jcraigk** [comment]
HAML is dead but ERB is very suboptimal as a templating language in 2020.  I suggest using Slim instead.
